File: list-00/c/exercise-00/exercise-00.c
// Somando dois numero inteiros

#include <stdio.h>

int main(){

	int num1,num2,sum;

	printf("Insira o primeiro valor: ");
	scanf("%i", &num1);
	printf("Insira o segundo valor: ");
	scanf("%i", &num2);
	
	sum = num1 + num2;

	printf("A soma dos dois valores é: %i\n", sum);

	return 0;

}

File: list-00/java/exercise00/exercise00.java
//Faça um programa que peça dois números inteiros e imprima a soma desses dois números

package exercise00;
import java.util.Scanner;

public class exercise00 {
    public static void main(String[] args){
        int num1,num2;
        
        Scanner numero = new Scanner(System.in);
        
        System.out.println("Insira um número: ");
        num1 = numero.nextInt();
        System.out.println("Insira mais um número: ");
        num2 = numero.nextInt();
        
        System.out.println("A soma dos números é: "+(num1 + num2));
        
    }
    
}
